odoo/addons/base/wizard/base_partner_merge.py

```diff
@@ -318,6 +318,9 @@ def _merge(self, partner_ids, dst_partner=None, extra_checks=True):
         self._update_reference_fields(src_partners, dst_partner)
         self._update_values(src_partners, dst_partner)
 
+        self.env.add_to_compute(dst_partner._fields['partner_share'], dst_partner)
+        dst_partner._recompute_recordset(['partner_share'])
```

Comment:
This line looks useless to me. Every record tags to recompute will be recompute at the end of request. No need to call it yourself.